app/templates/overview.hbs

```diff
@@ -0,0 +1,7 @@
+{{#nav/login-nav title="Welcome"}}
```

Comment:
login-nav disables the drawer and shows the login-options automaticly
If this is accessible even when you are logged-in, you should use "nav/index-nav" instead